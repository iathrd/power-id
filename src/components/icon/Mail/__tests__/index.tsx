import renderer from "react-test-renderer";
import Component from "../index";

it("renders corecly", () => {
  const tree = renderer.create(<Component />).toJSON();
  expect(tree).toMatchSnapshot();
});
