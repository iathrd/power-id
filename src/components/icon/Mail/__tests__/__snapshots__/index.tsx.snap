// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders corecly 1`] = `
<svg
  fill="none"
  height="24"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M4 4H20C21.1 4 22 4.9 22 6V18C22 19.1 21.1 20 20 20H4C2.9 20 2 19.1 2 18V6C2 4.9 2.9 4 4 4Z"
    stroke="#9B9B9B"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2.33333"
  />
  <path
    d="M22 6L12 13L2 6"
    stroke="#9B9B9B"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2.33333"
  />
</svg>
`;
